Makefile

```diff
@@ -60,14 +60,6 @@ else
 	@go test -mod=readonly $(ARGS) $(TEST_PACKAGES)
 endif
 
-build-docker-test:
-	@echo "--> Building docker image..."
-	@docker build -f Dockerfile.test -t peggo-test .
-
-docker-test:
-	@echo "--> Running tests in docker..."
-	@docker run peggo-test
```

Comment:
why removing this?

Reply:
it was not being used, the peggo-test also did not exist as well